templates/RED.html

```diff
@@ -22,6 +21,11 @@
                   <a onclick="viewSection('operations', '/campaign/operations')">operations</a>
                 </div>
               </div>
+              {% if errors|length > 0 %}
```

Comment:
if there are errors, what do you think about still showing the version (not errors.png) but ensure it is highlighted red instead of green? this seems like it'd be more consistent. thoughts?

Reply:
yea, we could do that, then it is just the background color changing for that section in the navbar.  Though if the error then is the core code the version will show 'None'.  So if that is acceptable I am good with that.

Reply:
pushed up a change to implement this and pushed up a change to show "no version" instead of 'None' as well, we can deliberate on if we like that or not..